src/components/scenes/MigrateWalletCompletionScene.tsx

```diff
@@ -168,6 +170,11 @@ const MigrateWalletCompletionComponent = (props: Props) => {
           let spendInfo: EdgeSpendInfo = {
             currencyCode: oldWallet.currencyInfo.currencyCode,
             spendTargets: [{ publicAddress: newPublicAddress }],
+            metadata: {
+              category: 'transfer',
```

Comment:
It needs to be `'Transfer'`, uppercase. We have always capitalized this, since the Airbitz days.